src/Generation/Generator/Renderer/Internal/ReturnType/Converter/InterfaceArray.cs

```diff
@@ -0,0 +1,14 @@
+namespace Generator.Renderer.Internal.ReturnType;
+
+internal class InterfaceArray : ReturnTypeConverter
```

Comment:
As this class handles "GLibPtrArray" please name it `InterfaceGLibPtrArray` instead of `InterfaceArray`. We need to keep this name available for the regular array implementations.